libs/template/presets/fulfillment/app.ts

```diff
@@ -62,6 +62,10 @@ export function fulfillmentFeatures(
               provide: ThemeMetaInitializer,
               useClass: PWAThemeMetaInitializer,
             },
+            {
+              provide: FeatureOptions,
+              useValue: { 'oryx-app': { colorMode: ColorMode.Light } },
```

Comment:
But can we change experience in patch? For me it looks like a minor and for me it looks like we need to wrap it with featureVersion.

Reply:
The experience does not change. It's a fix to me. We initially forced this behaviour with boilerplate, but realized that we should do it with feature set. I'd not overcomplicate it.

Reply:
Well, as long as it's in our module - it can change an experience as we're forcing some specific mode. But ok, it's not critical to me.